README.md

```diff
@@ -21,66 +44,50 @@ $ cmake .. -DCMAKE_INSTALL_PREFIX=../install
 $ cmake --build . --target install
 $ cd ..
 $ export CYCLONEDDS_HOME="$(pwd)/install"
-$ pip3 install --user git+https://github.com/eclipse-cyclonedds/cyclonedds-python
+$ pip3 install cyclonedds --no-binary cyclonedds
 ```
 
-If you want to install with development tools add the `[dev]` component to your installation like so:
+<!----><a name="installing-from-source-via-git"></a>
+### via git
+
+A full example installation of the quickest way to get started via git is shown below:
+
+```bash
+$ git clone https://github.com/eclipse-cyclonedds/cyclonedds
+$ cd cyclonedds && mkdir build install && cd build
+$ cmake .. -DCMAKE_INSTALL_PREFIX=../install
+$ cmake --build . --target install
+$ cd ..
+$ export CYCLONEDDS_HOME="$(pwd)/install"
+$ pip3 install git+https://github.com/eclipse-cyclonedds/cyclonedds-python
+```
+
+# Extra dependencies
+
+The `cyclonedds` package defines two sets of optional dependencies, `dev` and `docs`, used for developing `cyclonedds` and building the documentation, respectively. If you want to install with development tools add the component to your installation, for example:
 
 ```bash
 $ pip3 install --user "cyclonedds[dev] @ git+https://github.com/eclipse-cyclonedds/cyclonedds-python"
 ```
 
-Installation from a local git clone:
+Or when installing from a local git clone, which is recommended when developing or building the docs:
 
 ```bash
 $ cd /path/to/git/clone
-$ pip3 install --user .
-# or for development:
+# for development:
 $ pip3 install --user .[dev]
```

Comment:
I find I have to quote the options to prevent them from being interpreted as file name patterns (could well be a zsh-vs-bash thing). Perhaps it would be better to use
```suggestion
$ pip3 install --user ".[dev]"
```